### PR TITLE
Use moonbeam pallet_parachain_staking

### PR DESCRIPTION
**Pull request by asiniscalchi**
The Moonbeam `pallet_parachain_staking` remains largely unmodified, except for minimal adjustments necessary for compilation. 
All significant development efforts have been concentrated in the `ownership-chain/runtime/laos/src/configs/parachain_staking.rs` file.

### CODE REVIEW

File: ownership-chain/runtime/laos/src/fee.rs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	}
+}
+
+impl<R> OnUnbalanced<Credit<R::AccountId, pallet_balances::Pallet<R>>> for DealWithFees<R>
+where
+	R: pallet_balances::Config,
+{
+	// this is called from pallet_evm for Ethereum-based transactions
+	fn on_nonzero_unbalanced(_amount: Credit<R::AccountId, pallet_balances::Pallet<R>>) {
+		// TODO actually the fees are burned
```
Comment (magecnion) [inline ownership-chain/runtime/laos/src/fee.rs:24]: TODO

File: ownership-chain/runtime/laos/src/tests/mod.rs [hunk 2]
```diff
@@ -142,49 +142,3 @@ fn account_vests_correctly_over_time() {
...
-	new_test_ext().execute_with(|| assert!(ParachainStaking::inflation_enabled() == false));
-}
-
-#[test]
-fn collator_rewards_account_is_not_set_by_default() {
-	new_test_ext().execute_with(|| assert!(ParachainStaking::collator_rewards_account().is_none()));
-}
-
-#[test]
-fn fees_go_to_collator_rewards_account() {
```
Comment (magecnion) [inline ownership-chain/runtime/laos/src/tests/mod.rs:157]: what does happen with fees now?

File: ownership-chain/runtime/laos/src/fee.rs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+/// transaction fees), the amount is transferred to the collator rewards account.
+pub struct DealWithFees<R>(PhantomData<R>);
+
+type NegativeImbalanceOfBalances<T> = pallet_balances::NegativeImbalance<T>;
+impl<R> OnUnbalanced<NegativeImbalanceOfBalances<R>> for DealWithFees<R>
+where
+	R: pallet_balances::Config + pallet_parachain_staking::Config,
+{
+	fn on_nonzero_unbalanced(_amount: NegativeImbalanceOfBalances<R>) {
+		// TODO actually the fees are burned
```
[the remark this answers is not in the document]
Comment (asiniscalchi) [reply ownership-chain/runtime/laos/src/fee.rs:14]: I am implementing it here : https://github.com/freeverseio/laos/pull/408

File: ownership-chain/runtime/laos/src/fee.rs (hunk 1 above)
Comment (asiniscalchi) [reply ownership-chain/runtime/laos/src/fee.rs:24]: I am implementing it here : https://github.com/freeverseio/laos/pull/408

File: ownership-chain/runtime/laos/src/tests/mod.rs (hunk 2 above)
Comment (asiniscalchi) [reply ownership-chain/runtime/laos/src/tests/mod.rs:157]: I am implementing it here : https://github.com/freeverseio/laos/pull/408

**Review by dastanfv — APPROVED** [review]

**Review by magecnion — APPROVED** [review]

**Pull request merged**